include/warehouse_ros_mongo/query_results.h

```diff
@@ -49,13 +49,13 @@ namespace warehouse_ros_mongo
 {
 // To avoid some const-correctness issues we wrap Mongo's returned auto_ptr in
```

Comment:
This looks weird. I have filed a PR (https://github.com/christian-rauch/warehouse_ros_mongo/pull/1) to clean this up.

Reply:
Merged.